code/modules/tgui/states/physical.dm

```diff
@@ -19,3 +19,6 @@
 
 /mob/living/silicon/physical_can_use_topic(src_object)
 	return max(UI_UPDATE, shared_living_ui_distance(src_object)) // Silicons can always see.
+
+/mob/living/silicon/ai/physical_can_use_topic(src_object)
+	return UI_CLOSE // AIs are not physical.
```

Comment:
That is disappointing. Canisters have an electronic interface with physical controls, and AIs could never *use* the canisters, but could still read them. Oh well :(